juristische-zitierweise-schweizer.csl

```diff
@@ -0,0 +1,391 @@
+                <text variable="title-short"/>
+              </if>
+            </choose>
+            <text variable="locator"/>
+          </group>
+        </if>
+        <else-if type="book">
+          <group delimiter=", ">
+            <text macro="autor-editor-note"/>
+            <choose>
+              <if match="any" variable="title-short">
+                <text variable="title" form="short"/>
+              </if>
+            </choose>
+            <text variable="locator"/>
+          </group>
+        </else-if>
+        <else-if type="thesis">
+          <group delimiter=", ">
+            <text macro="autor-editor-note"/>
+            <choose>
+              <if match="any" variable="title-short">
+                <text variable="title" form="short"/>
+              </if>
+            </choose>
+            <text variable="locator"/>
+          </group>
+        </else-if>
+        <else-if type="chapter">
+          <group delimiter=", ">
+            <text macro="author-note"/>
+            <choose>
+              <if match="any" variable="title-short">
+                <text variable="title" form="short" prefix="in: "/>
+              </if>
+            </choose>
+            <text variable="locator"/>
+          </group>
+        </else-if>
+        <else-if type="entry-encyclopedia">
+          <text variable="title" suffix="-" form="short"/>
+          <text macro="author-note" suffix=", "/>
+          <text variable="title" form="long" suffix=" "/>
+          <text variable="locator"/>
+        </else-if>
+        <else-if type="legal_case" match="any">
+          <group delimiter=", ">
+            <choose>
+              <if match="any" variable="title">
+                <text variable="title"/>
+              </if>
+              <else>
+                <group delimiter=" ">
+                  <text variable="authority"/>
+                  <choose>
+                    <if match="any" variable="abstract">
+                      <text variable="number" suffix=", "/>
+                      <text variable="abstract"/>
+                    </if>
+                    <else>
+                      <text variable="number"/>
+                    </else>
+                  </choose>
+                  <group delimiter="">
+                    <text macro="issued-date" prefix="vom "/>
+                    <choose>
+                      <if match="any" variable="container-title">
+                        <text variable="container-title" prefix=", "/>
+                        <text variable="volume" prefix=" "/>
+                        <text variable="page" prefix=", "/>
+                      </if>
+                    </choose>
+                  </group>
+                  <text variable="title" form="short" font-style="italic" prefix="&#8211; "/>
+                </group>
+              </else>
+            </choose>
+            <text variable="locator"/>
+          </group>
+          <text variable="note" prefix=" "/>
```

Comment:
there's not way to find an alternative here? "note" is very erratically mapped from different CSL applications and we generally try to avoid it in citation styles.